src/main/java/frc/robot/subsystems/TestableSubsystem.java

```diff
@@ -1,5 +1,9 @@
 package frc.robot.subsystems;
 
-public interface TestableSubsystem extends MovableSubsystem, ReadableSubsystem {
+import frc.robot.constants.RobotConstants.TesterConstants;
 
+public interface TestableSubsystem extends MovableSubsystem, ReadableSubsystem {
+    default void test() {
+        move(TesterConstants.MOVE_POWER);
```

Comment:
not every motor should receive the same power. maybe

Reply:
you're right I didn't realize that but change the constant name to DEFAULT_ MOVE_POWER